src/Updater/PuliStrategy.php

```diff
@@ -138,4 +139,29 @@ public function getStability()
     {
         return $this->stability;
     }
+
+    /**
+     * Get a remote file content securely using composer/ca-bundle to find the local ca-file.
+     *
+     * @param string $remoteUrl
+     * @return string
+     */
+    private function secureFileGetContents($remoteUrl)
+    {
+        $options = array(
+            'http' => array(
+                'method' => 'GET'
+            )
+        );
+
+        $caPath = CaBundle::getSystemCaRootBundlePath();
+
+        if (is_dir($caPath)) {
+            $options['ssl']['capath'] = $caPath;
+        } else {
+            $options['ssl']['cafile'] = $caPath;
+        }
+
+        return file_get_contents($remoteUrl, false, stream_context_create($options));
```

Comment:
I do not think so.. 

From the docs: 

> Defines a humbug_get_contents() function that will transparently call file_get_contents(), except for HTTPS URIs where it will inject a context configured to enable secure SSL/TLS requests on all versions of PHP less than 5.6.

> All versions of PHP below 5.6 not only disable SSL/TLS protections by default, but have most other default options set insecurely. This has led to the spread of insecure uses of file_get_contents() to retrieve HTTPS resources. For example, PHAR files or API requests. Without SSL/TLS protections, all such requests are vulnerable to Man-In-The-Middle attacks where a hacker can inject a fake response, e.g. a tailored php file or json response.